app/controllers/admin/dashboard_controller.rb

```diff
@@ -0,0 +1,7 @@
+class Admin::DashboardController < Admin::AdminController
+  def dashboard
```

Comment:
When pairing with linesarefuzzy, Fuzzy requested that this action not be index or show, but instead a custom action named dashboard.